FCCee/compact/FCCee_o1_v04/OuterTracker_o2_v06_02.xml

```diff
@@ -180,27 +180,30 @@
         
          
 	<detector name="OuterTrackerEndcapSupport" type="TrackerEndcapSupport_o1_v01" reflect="true" region="OuterTrackerEndcapRegion">
-		   <envelope>
-		<shape type="Assembly"/>
-	   </envelope>
-
-	   <layer id="0" inner_r="OuterTracker_inner_radius" inner_z="OuterTracker_Endcap_z_0+1*cm" outer_r="OuterTracker_Endcap_outer_radius" vis="SupportVis">
-		   <slice material="CarbonFiber" thickness="0.365/3.5959*cm" />
-	   </layer>
-	   <layer id="1" inner_r="OuterTracker_inner_radius" inner_z="OuterTracker_Endcap_z_1-1*cm" outer_r="OuterTracker_Endcap_outer_radius" vis="SupportVis">
-		   <slice material="CarbonFiber" thickness="0.365/3.5959*cm" />
-	   </layer>
-	   <layer id="2" inner_r="OuterTracker_inner_radius" inner_z="OuterTracker_Endcap_z_2+1*cm" outer_r="OuterTracker_Endcap_outer_radius" vis="SupportVis">
-		   <slice material="CarbonFiber" thickness="0.365/3.5959*cm" />
-	   </layer>
-	   <layer id="3" inner_r="OuterTracker_inner_radius" inner_z="OuterTracker_Endcap_z_3-1*cm" outer_r="OuterTracker_Endcap_outer_radius" vis="SupportVis">
-		   <slice material="CarbonFiber" thickness="0.365/3.5959*cm" />
-	   </layer>
-	   <layer id="4" inner_r="OuterTracker_Barrel_radius_0-1*cm" inner_z="OuterTracker_Barrel_half_length+2*cm" outer_r="OuterTracker_outer_radius" vis="OuterTrackerInterlinkVis">
-		   <slice material="CarbonFiber" thickness="0.6/3.5959*cm" />
-	   </layer>
+	  <envelope>
+	    <shape type="Assembly"/>
+	  </envelope>
 
+	  <layer id="0" inner_r="OuterTracker_inner_radius" inner_z="OuterTracker_Endcap_z_0+1*cm" outer_r="OuterTracker_Endcap_outer_radius" vis="SupportVis">
+	    <slice material="CarbonFiber" thickness="0.365/3.5959*cm" />
+	  </layer>
+	  <layer id="1" inner_r="OuterTracker_inner_radius" inner_z="OuterTracker_Endcap_z_1-1*cm" outer_r="OuterTracker_Endcap_outer_radius" vis="SupportVis">
+	    <slice material="CarbonFiber" thickness="0.365/3.5959*cm" />
+	  </layer>
+	  <layer id="2" inner_r="OuterTracker_inner_radius" inner_z="OuterTracker_Endcap_z_2+1*cm" outer_r="OuterTracker_Endcap_outer_radius" vis="SupportVis">
+	    <slice material="CarbonFiber" thickness="0.365/3.5959*cm" />
+	  </layer>
+	  <layer id="3" inner_r="OuterTracker_inner_radius" inner_z="OuterTracker_Endcap_z_3-1*cm" outer_r="OuterTracker_Endcap_outer_radius" vis="SupportVis">
+	    <slice material="CarbonFiber" thickness="0.365/3.5959*cm" />
+	  </layer>
+	  <layer id="4" inner_r="OuterTracker_Barrel_radius_0-1*cm" inner_z="OuterTracker_Barrel_half_length+2*cm" outer_r="OuterTracker_outer_radius_1-0.5*cm" vis="OuterTrackerInterlinkVis">
```

Comment:
```suggestion
	  <layer id="4" inner_r="OuterTracker_Barrel_radius_0-1*cm" inner_z="OuterTracker_Barrel_half_length+2*cm" outer_r="OuterTracker_outer_radius-0.5*cm" vis="OuterTrackerInterlinkVis">
```

Reply:
done

Reply:
This was actually correct before my suggestion (#280)